src/Schema/Column/ColumnFactoryInterface.php

```diff
@@ -26,14 +27,14 @@
  *     defaultValueRaw?: string|null,
  *     dimension?: positive-int,
  *     extra?: string|null,
- *     fromResult?: bool,
  *     primaryKey?: bool,
  *     name?: string|null,
  *     notNull?: bool,
  *     reference?: ForeignKey|null,
  *     scale?: int|null,
  *     schema?: string|null,
  *     size?: int|null,
+ *     source?: ColumnInfoSource::*,
```

Comment:
Why is this info needed? It seems that `fromResult` and `source` are never used currently.

Reply:
This can be useful when a user defines his own method/closure to create the column object in column factory.